pkgs/servers/tigerbeetle/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "tigerbeetle";
+  version = "unstable-2022-09-30";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetledb";
+    repo = pname;
+    rev = "2023-02-13-weekly";
+    sha256 = "sha256-OcVLFnzW140Drx/jQ6p3eW1ZDGXrRSM6eQ47L+Ub54A=";
+  };
+
+  buildInputs = [ zig ];
```

Comment:
`zig` is a build tool here and for cross-compilation you would probably want to use not host's, but build's tool. The usual way to do it is to use `nativeBuildInputs`: `nativeBuildInputs = [ zig ];`.

The rule of thumb is: if the tool is used at build tile it goes to `nativeBuildInputs`, if the tool is used at runtime (after package is installed) it belongs to `buildInputs`.